portfolio/src/main/webapp/style.css

```diff
@@ -60,16 +130,20 @@ h1, h2 {
   -webkit-transform: translate(-50%, -50%);
   -ms-transform: translate(-50%, -50%);
   color: white;
-  font-size: 9px;
+  font-family: Garamond;
+  font-size: 12px;
   left: 50%;
   position: absolute;
   text-align: center;
   top: 50%;
   transform: translate(-50%, -50%);
 }
 
-.proj-container {
+.proj-buttons {
+  left: 15%;
+  margin-top: 20px;
   position: relative;
+  padding: 10px;
 }
 
 .slide {
```

Comment:
can this style be deleted?

Reply:
I believe this style ensures that each "slide" element is initially hidden by default in the image slideshow.